server/webhook.go

```diff
@@ -492,8 +492,14 @@ func (s *Server) processPushEvent(ctx context.Context, repositoryList []*api.Rep
 		return nil, nil
 	}
 
+	repo := repositoryList[0]
```

Comment:
It is possible to panic here and we should do a length check, if it would never happen, we should add a code comment explaining why (but I don't think the latter is a good idea since it creates unnecessary coupling with the call sites).

Reply:
This function expects the repositoryList to be non-empty. The caller should check it. Since it's a private function, it's OK IMO.

Reply:
>This function expects the repositoryList to be non-empty.

This is the exact implicit coupling I would worry about :)

Reply:
>> This function expects the repositoryList to be non-empty.
>
> This is the exact implicit coupling I would worry about :)

You are right. But panic in the Bytebase server is always a bad idea. There's a tradeoff of how much defensive coding we should do.

Reply:
Just do it whenever we can!

Reply:
Haha, alright.